internal/wire/parse.go

```diff
@@ -248,8 +248,8 @@ type Field struct {
 // env is nil or empty, it is interpreted as an empty set of variables.
 // In case of duplicate environment variables, the last one in the list
 // takes precedence.
-func Load(ctx context.Context, wd string, env []string, patterns []string) (*Info, []error) {
-	pkgs, errs := load(ctx, wd, env, patterns)
+func Load(ctx context.Context, wd string, env []string, patterns []string, tags string) (*Info, []error) {
```

Comment:
small nit: let's put `tags` before `patterns`, since it's more closely related to the environment.

```suggestion
func Load(ctx context.Context, wd string, env []string, tags string, patterns []string) (*Info, []error) {
```